ui/appui-react/src/appui-react/stagepanels/StagePanelDef.tsx

```diff
@@ -182,17 +216,17 @@ export class StagePanelDef extends WidgetHost {
 
     if (!config)
       return;
-    this._size = config.size;
+    this.size = config.size;
```

Comment:
Assign to field to avoid firing change events when initializing?

```suggestion
    this._size = config.size;
```